src/03-modality-agnostic-files.md

```diff
@@ -197,6 +201,15 @@ for them:
     -   for "ambidextrous", use one of these values: `ambidextrous`, `a`, `A`,
         `AMBIDEXTROUS`, `Ambidextrous`
 
+-   `strain`: string value indicating the strain of the species
```

Comment:
Examples for each of these would be useful.

Reply:
Just to clarify, do you mean example directly in the description, like above for handedness with `ambidextrous`, `a`, `A`, etc, or an example of `particpants.tsv` for an animal.

I did not change the example of `participants.tsv` below which is an example for human. I thought having complete examples for both human and animal would maybe be too much.

Reply:
perhaps a few examples from: https://www.jax.org/jax-mice-and-services/find-and-order-jax-mice/most-popular-jax-mice-strains

Reply:
Examples added in b451671.